create.ps1

```diff
@@ -1,378 +1,222 @@
-#####################################################
+#################################################
 # HelloID-Conn-Prov-Target-SDBHR-Create
-#
-# Version: 1.0.0
-#####################################################
-# Initialize default values
-$c = $configuration | ConvertFrom-Json
-$p = $person | ConvertFrom-Json
-$success = $false # Set to false at start, at the end, only when no error occurs it is set to true
-$auditLogs = [System.Collections.Generic.List[PSCustomObject]]::new()
-
-# Set TLS to accept TLS, TLS 1.1 and TLS 1.2
-[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls -bor [Net.SecurityProtocolType]::Tls11 -bor [Net.SecurityProtocolType]::Tls12
+# Correlate to account
+# PowerShell V2
+#################################################
+# Enable TLS1.2
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
 
 # Set debug logging
-switch ($($c.isDebug)) {
+switch ($actionContext.Configuration.isDebug) {
```

Comment:
If making chances maybe remove isDebug and replace all write-verbose for write-information?
Also this can be removed then:
$VerbosePreference = "SilentlyContinue"
$InformationPreference = "Continue"
$WarningPreference = "Continue"